lib/tether/tether.go

```diff
@@ -424,6 +430,20 @@ func (t *tether) processSessions() error {
 					extraconfig.EncodeWithPrefix(t.sink, session, extraconfig.CalculateKeys(t.config, fmt.Sprintf("%s.%s", session.extraconfigKey, id), "")[0])
 					log.Warnf("Re-launching process for session %s (count: %d)", id, session.Diagnostics.ResurrectionCount)
 					session.Cmd = *restartableCmd(&session.Cmd)
+
```

Comment:
This is wrong and will break tether. 

You should not launch the session by yourself like this and instead only call loggingLocked and let the next block to handle launching them in parallel.